Analysis/include/SampleDescriptor.h

```diff
@@ -39,14 +41,35 @@ struct AnalyzerSetup {
     std::string plot_cfg, plot_page_opt, unc_cfg;
     JetOrdering jet_ordering;
     std::map<Channel, std::vector<std::string>> trigger;
-
     std::map<SelectionCut,analysis::EllipseParameters> massWindowParams;
+    std::map<std::string, std::vector<std::string>> limit_setup_raw;
 
     bool IsSignal(const std::string& sample_name) const
     {
         const auto iter = std::find(signals.begin(), signals.end(), sample_name);
         return iter != signals.end();
     }
+
+    void CreateLimitSetups()
+    {
+        std::map<EventCategory, std::string> second_map;
```

Comment:
why do you need define second_map rather than access it directly through limit_setup?